hax-types/src/cli_options.rs

```diff
@@ -328,6 +328,10 @@ pub struct BackendOptions {
 
     #[command(flatten)]
     pub translation_options: TranslationOptions,
+
+    /// Where to put the output files.
```

Comment:
Maybe add a "defaults to `<crate folder>/proofs/<backend>/extraction`" or something along those lines?

Reply:
Done.